modules/special-rules/pages/ripper-tentacles.adoc

```diff
@@ -11,4 +11,6 @@ Apply these rules where your War Engine unit is armed with {ripper-tentacles}:
 
 * In close combat with opposing war engines, choose one {close-combat-weapons} system -- the {ripper-tentacles} wrap around and trap that weapon system, which means that the opposing War Engine unit cannot use it in the close combat.
 
-+[TODO: Presumably, if in close combat with multiple opposing war engines, you can only choose one CCW from among the opponents, not one each. Also, as part of a wider question, should weapon arcs apply to {ripper-tentacles} and {close-combat-weapons}?]+
\ No newline at end of file
++[TODO: Presumably, if in close combat with multiple opposing war engines, you can only choose one CCW from among the opponents, not one each. Also, as part of a wider question, should weapon arcs apply to {ripper-tentacles} and {close-combat-weapons}?]+
++
+[REPLY IJW: The original is all written in the singular, so yes I think it's always going to be a single CCW. Weapon arcs - why wouldn't they apply?]+
```

Comment:
"Why wouldn't they apply?" Well, I'm sure that people could rationalise it in either direction. But my point is that there is nothing explicit to make it clear whether or how fire arcs affect these weapons either as attacker or defender, and so my sub-text is **should** we clarify, and how?

And how complex should we get to cover edge cases? Eg, in a two-on-one combat, if the enemy war engine has a CCW on the side that is farthest from the Ripper Tentacles, can the tentacles still trap the CCW so that the enemy war engine can't use it against the other friendly war engine that is inside the arc of the CCW...?

Reply:
> "Why wouldn't they apply?" Well, I'm sure that people could rationalise it in either direction. But my point is that there is nothing explicit to make it clear whether or how fire arcs affect these weapons either as attacker or defender, and so my sub-text is **should** we clarify, and how?

`shooting-with-war-engines.adoc > War engine fire arcs` should already clarify it, I purposefully didn't use any of GW's 'can fire in an arc' phrasing, to make sure that it would cover all types of weapon:

> Each weapon mount on a war engine has a fire arc, and can only be used against targets within that fire arc.

I don't think there's a substantial balance issue with CCWs and Ripper Tentacles being restricted, background-wise you should be able to blind-side an enemy Titan by avoiding it's CCW, and it keeps things consistent. Especially as most CCWs have a gun built in!

That said, a few of the war engines could do with expanded fire arcs, the main culprit being the Gargant which gets a triple-whammy of paying 'full' cost for three weapons, dies on a 10+ on the Critical Damage Table, and only has forward 90º arcs on all three weapons.

> And how complex should we get to cover edge cases? Eg, in a two-on-one combat, if the enemy war engine has a CCW on the side that is farthest from the Ripper Tentacles, can the tentacles still trap the CCW so that the enemy war engine can't use it against the other friendly war engine that is inside the arc of the CCW...?
I think it's covered - the target is the 'opposing war engine' and the target is in fire arc.

But if you think that's not clear enough, we can add an example, or wait to see if anyone asks about it.

Reply:
I had another look at the original GW text for CCWs and Ripper Tentacles, and they use 'War engines armed with close combat weapons' and 'When in close combat with another war engine', which could infer possession is enough, and that there should be no fire arcs.

But...

Looking at the datasheets:
The Hierophant's Ripper Tentacles are listed with a Front arc, and has no ranged component.
The Subjugator's CCW is listed with a Front arc, and has no ranged component.

So I think it's safe to default to arcs applying to _all_ weapons.

Reply:
> I'm mildly attracted to the idea that perhaps Ripper Tentacles ought not to be able to trap a CCW that is on the far side of the opposing war engine (maybe only able to trap it if their arcs overlap). But, given that the target is the "opposing war engine" as you mention, it does seem this idea would be a rules change, strictly speaking.

Yeah, I know what you mean, but only looking at the user's arc should keep things simpler.

Reply:
>I think it's covered - the target is the 'opposing war engine' and the target is in fire arc.

I see what you mean, and hopefully your efforts to firm up description of weapon arcs should eliminate the temptation to 'clarify' in topics like this one.